sherlock/__main__.py

```diff
@@ -11,18 +11,10 @@
 
 
 if __name__ == "__main__":
-    # Checking if the user is using the correct version of Python
-    # Reference:
-    #  If Python version is 3.6.5
-    #               major --^
-    #               minor ----^
-    #               micro ------^
-    major = sys.version_info[0]
-    minor = sys.version_info[1]
+    """Check if the user is using the correct version of Python"""
```

Comment:
You should probably use `#` for comment and not `"""`.

Reply:
Okay, I've changed it back to #.